### PR TITLE
#161967092 users should be able to create articles

### PR DESCRIPTION
**Pull request by jillo-abdullahi**
**What does this PR do?**
-  Implements user ability to create articles and view created article.

**Description of tasks to be completed?**
- Authenticated users can create a new article.
- The created article can then be viewed by the user.

**How should this be manually tested?**

- Clone the repo and cd into the project folder
- Switch to the appropriate branch with `gco ft-create-article-161967092`
- Run `npm install` to update your packages.
- Run tests with `npm test`
- Fire up the server with `npm start`
- Navigate to `http://localhost:3000/create_article`

**What are the relevant pivotal tracker stories?**
- [#161967092](https://www.pivotaltracker.com/story/show/161967092)

**Checklist (edited)**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

**Screenshots**

screenshot 2019-01-17 at 15 09 44

screenshot 2019-01-17 at 15 12 51

### CODE REVIEW

**Review by Bryan-Cee — APPROVED** [review]
👍 Works fine

**Review by Darius-Ndubi — APPROVED** [review]
👍

**Review by davishooly — APPROVED** [review]

File: src/components/__tests__/articleActions.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  CREATE_ARTICLE_SUCCESS
+} from "../../actions/actionTypes";
+import {
+  createArticleSuccess,
+  getSpecificArticle
+} from "../../actions/articlesActions";
+
+describe("action creators", () => {
+  it("should dispatch CREATE_ARTICLE_SUCCESS type", () => {
+    expect(createArticleSuccess({}).type).toEqual(CREATE_ARTICLE_SUCCESS);
```
Comment (justmesam) [inline src/components/__tests__/articleActions.test.js]: use a mock store to dispatch the actions.

**Review by justmesam — CHANGES_REQUESTED** [review]

File: src/components/__tests__/articleActions.test.js (hunk 1 above)
Comment (jillo-abdullahi) [reply src/components/__tests__/articleActions.test.js]: On it!

Comment (jillo-abdullahi) [reply src/components/__tests__/articleActions.test.js]: This is done!

**Review by justmesam — APPROVED** [review]

**Comment by justmesam** [comment]
LGTM 👍

**Review by denomash — APPROVED** [review]
👍

**Pull request merged**